storage/engine/rocksdb_test.go

```diff
@@ -136,51 +138,61 @@ func TestRocksDBCompaction(t *testing.T) {
 func setupMVCCScanData(numVersions, numKeys, valueBytes int, b *testing.B) (*RocksDB, *stop.Stopper) {
 	loc := fmt.Sprintf("mvcc_scan_%d_%d_%d", numVersions, numKeys, valueBytes)
 
-	exists := true
-	if _, err := os.Stat(loc); os.IsNotExist(err) {
-		exists = false
-	}
-
-	log.Infof("creating mvcc data: %s", loc)
 	const cacheSize = 8 << 30 // 8 GB
-	stopper := stop.NewStopper()
-	rocksdb := NewRocksDB(roachpb.Attributes{Attrs: []string{"ssd"}}, loc, cacheSize, stopper)
-	if err := rocksdb.Open(); err != nil {
-		b.Fatalf("could not create new rocksdb db instance at %s: %v", loc, err)
-	}
 
-	if exists {
-		return rocksdb, stopper
-	}
+	if _, err := os.Stat(loc); os.IsNotExist(err) {
+		log.Infof("creating mvcc data: %s", loc)
 
-	rng, _ := randutil.NewPseudoRand()
-	keys := make([]roachpb.Key, numKeys)
-	nvs := make([]int, numKeys)
-	for t := 1; t <= numVersions; t++ {
-		walltime := int64(5 * t)
-		ts := makeTS(walltime, 0)
-		batch := rocksdb.NewBatch()
-		for i := 0; i < numKeys; i++ {
-			if t == 1 {
-				keys[i] = roachpb.Key(encoding.EncodeUvarint([]byte("key-"), uint64(i)))
-				nvs[i] = rand.Intn(numVersions) + 1
-			}
-			// Only write values if this iteration is less than the random
-			// number of versions chosen for this key.
-			if t <= nvs[i] {
-				value := roachpb.MakeValueFromBytes(randutil.RandBytes(rng, valueBytes))
-				value.InitChecksum(keys[i])
-				if err := MVCCPut(batch, nil, keys[i], ts, value, nil); err != nil {
-					b.Fatal(err)
+		stopper := stop.NewStopper()
+		rocksdb := NewRocksDB(roachpb.Attributes{Attrs: []string{"ssd"}}, loc, cacheSize, stopper)
+		if err := rocksdb.Open(); err != nil {
+			b.Fatal(err)
+		}
+
+		rng, _ := randutil.NewPseudoRand()
+		keys := make([]roachpb.Key, numKeys)
+		nvs := make([]int, numKeys)
+		for t := 1; t <= numVersions; t++ {
+			walltime := int64(5 * t)
+			ts := makeTS(walltime, 0)
+			batch := rocksdb.NewBatch()
+			for i := 0; i < numKeys; i++ {
+				if t == 1 {
+					keys[i] = roachpb.Key(encoding.EncodeUvarint([]byte("key-"), uint64(i)))
+					nvs[i] = rand.Intn(numVersions) + 1
+				}
+				// Only write values if this iteration is less than the random
+				// number of versions chosen for this key.
+				if t <= nvs[i] {
+					value := roachpb.MakeValueFromBytes(randutil.RandBytes(rng, valueBytes))
+					value.InitChecksum(keys[i])
+					if err := MVCCPut(batch, nil, keys[i], ts, value, nil); err != nil {
+						b.Fatal(err)
+					}
 				}
 			}
+			if err := batch.Commit(); err != nil {
+				b.Fatal(err)
+			}
+			batch.Close()
 		}
-		if err := batch.Commit(); err != nil {
-			b.Fatal(err)
-		}
-		batch.Close()
+		rocksdb.CompactRange(nil, nil)
+		stopper.Stop()
+	}
+
+	locDirty := loc + "_dirty"
```

Comment:
I was thinking you would only make the copy in `BenchmarkMVCCDeleteRange`. Performing the copy for every benchmark seems excessive.